content/articles/transacting-erc20-tokens-brownie/index.md

```diff
@@ -0,0 +1,351 @@
+    return my_token
+
+def main():
+    my_token = deploy_token()
+    quiz_reward = deploy_quiz_reward()
+    account_0 = accounts[0]
+    account_1 = accounts[1]
+    account_2 = accounts[2]
+    get_token_balance(account_0.address)
+
+    send_token(account_1,Web3.toWei(7_000_000,"ether"))
+    get_token_balance(account_0.address)
+
+    set_allowance(quiz_reward.address,Web3.toWei(6_000_000,"ether"))
+
+    quiz_reward.answerQuestion(11,account_2,my_token)
+    get_token_balance(account_0.address)
+    get_token_balance(account_1.address)
+    get_token_balance(account_2.address)
+    get_token_balance(quiz_reward.address)
+
+def get_token_balance(account_address):
+    my_token = MyToken[-1]
+    print(f"The account {account_address} has balance {my_token.balanceOf(account_address)}")
+
+def send_token(receiver_address,amount):
+    my_token = MyToken[-1]
+    my_token.transfer(receiver_address,amount)
+    get_token_balance(receiver_address)
+    
+def set_allowance(spender_address,amount):
+    my_token = MyToken[-1]
+    my_token.increaseAllowance(spender_address,amount)
+
+def deploy_quiz_reward():
+    quiz_reward = QuizReward.deploy({"from":accounts[0]})
+    return quiz_reward
+
+main()
+```
+
+Finally, run:
+```python
+brownie run transact_token
+```
+
+You will get this output:
+```
+...
+Transaction sent: 0xb59238b8cda7af32dff1780e0e8c665a34da0df18b9994a362a161f8601ed723
+  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 9
+  QuizReward.answerQuestion confirmed   Block: 10   Gas used: 62723 (0.52%)
+
+The account 0x66aB6D9362d4F35596279692F0251Db635165871 has balance 13000000000000000000000000
+The account 0x33A4622B82D4c04a53e170c638B944ce27cffce3 has balance 7000000000000000000000000
+The account 0x0063046686E46Dc6F15918b61AE2B121458534a5 has balance 1000000000000000000000000
+The account 0x9E4c14403d7d9A8A782044E86a93CAE09D7B2ac9 has balance 0
+Terminating local RPC client...
+```
+
+Clearly, the smart contract was able to transfer 1000000 tokens to the account as requested. It can also be observed that the smart contract has no tokens it possesses. It is only allowed to use the tokens in the founder's account. 
+
+### Conclusion
+To view all the ERC20 functions inherited from the OpenZeppelin library, follow this link:[OpenZeppelin ERC20](https://docs.openzeppelin.com/contracts/2.x/api/token/erc20)
+
+There are a few functions which have not been covered here. For example:
+
+`_burn`. This is the opposite of `_mint`. It destroys the tokens in an account and reduces the total supply circulating.
+
+`decreaseAllowance`. This is the opposite of `increaseAllowance`. It reduces the tokens that an address is allowed to spend on an account.
+
+With the basics in place, you can easily adapt these functions into your code without a hassle.
+
+ERC20 tokens are very useful and have found applications in a variety of areas.
+Hopefully this article shed some light on how to deploy them and how to use them in transactions.
```

Comment:
Rephrase your conclusion, connect the sentences and explain to the reader what he gained from this tutorial.
Add a further reading section.